cmds/truncate/truncate.go

```diff
@@ -68,9 +66,9 @@ func main() {
 			}
 		}
 
-		final := want // base case
-		if (*sizeStr)[0] == '+' || (*sizeStr)[0] == '-' {
-			final = st.Size() + want // in case of '-', want is already negative
+		final := size.Value // base case
```

Comment:
Does every program that uses unit have to do this if? It seems very inconvenient. What about people who forget to do this, will the error be mysterious?

Reply:
would it make sense just to change the arg so that if s[0] is not +, then s = "+" + s?

Reply:
Sorry, I don't get that one. Now the only thing you have to do is get the size.Value and be done. Maybe you want to check if it is <0 and fail in that case. Checking the ExplicitSign is specific to truncate as there it makes a difference whether the value is "+23" or "23" (information usually just lost if you strconv a value starting with a '+'). So I don't see the complication here, use size.Value and be done, it contains the correct size in bytes, negative or positive. But if you have a weird command like truncate, then you want to check the ExplicitSign case.

Maybe I got you wrong, but I don't see the inconvenience. Previously it was inconvenient and someone had to check s[0].